gr-qtgui/python/qtgui/distanceradar.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+from PyQt5 import QtWidgets
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.figure import Figure
```

Comment:
so matplotlib is an optional dependency, and we should check that it is installed before using it. 

See the "Python modules" section here: https://github.com/gnuradio/greps/blob/master/grep-0001-coding-guidelines.md

So these should be wrapped with a try / except like here: https://github.com/gnuradio/gnuradio/blob/master/gnuradio-runtime/examples/volk_benchmark/volk_plot.py#L10-L15

This might be worth more discussion if we should just make matplotlib a required dep, but this should still be wrapped up.

Reply:
This is worth discussing.